.gitignore

```diff
@@ -1,5 +1,5 @@
 node_modules
 /lib
 jest.env.js
-.idea
-yarn-error.log
\ No newline at end of file
+.ideayarn-error.log
```

Comment:
I guess it is mistake? Is there any reason why you've removed `yarn-error.log` and added `ideayarn-error.log` instead?

Reply:
Looks like it's accidentally typed key.